### PR TITLE
commands: submit_tuxbuild: enable duration data in ALLOWED_METADATA

### PR DESCRIPTION
**Pull request by roxell**
ALLOWED_METADATA doesn't save 'duration'. Duration can be used to track
the build time.

Signed-off-by: Anders Roxell <anders.roxell@linaro.org>

### CODE REVIEW

**Comment by chaws** [comment]
Nice stuff! Perhaps you could also send duration as metric? 
Just add 
```python
metrics.update({'build/duration': build["duration"]}
```
Right below this line: https://github.com/Linaro/squad-client/blob/master/squad_client/commands/submit_tuxbuild.py#L162

The "build/duration" is just a suggestion.

File: squad_client/commands/submit_tuxbuild.py [hunk 1]
```diff
@@ -160,6 +161,7 @@ def run(self, args):
 
             tests = {test_name: test_status}
             metrics = {test_name + '-warnings': warnings_count}
+            metrics.update({'build/duration': build["duration"]}
```
Comment (chaws) [inline squad_client/commands/submit_tuxbuild.py:164]: Ops, you forgot to close it :)

Comment (roxell) [reply squad_client/commands/submit_tuxbuild.py]: done

**Comment by chaws** [comment]
Make metric name contain compile + defconfig + env

**Comment by chaws** [comment]
LGTM, there's still one failing test. I can fix that one out. I'll let you know when I roll it out

**Pull request merged**